installing/installing_azure_stack_hub/installing-azure-stack-hub-default.adoc

```diff
@@ -0,0 +1,58 @@
+[id="installing-azure-stack-hub-default"]
+= Installing a cluster quickly on Azure Stack Hub
+include::modules/common-attributes.adoc[]
+:context: installing-azure-stack-hub-default
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
+Microsoft Azure Stack Hub that uses the default configuration options. However, you must manually configure the `install-config.yaml` file to specify values that are specific to Azure Stack Hub.
+
+[NOTE]
+====
+While you can select `azure` when using the installation program to deploy a cluster using installer-provisioned infrastructure, this option is only supported for the Azure Public Cloud.
+====
+
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+* You xref:../../installing/installing_azure_stack_hub/installing-azure-stack-hub-account.adoc#installing-azure-stack-hub-account[configured an Azure Stack Hub account] to host the cluster.
+* If you use a firewall, you xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured it to allow the sites] that your cluster requires access to.
+* You put the Cloud Credential Operator (COO) into xref:../../installing/installing_azure_stack_hub/manually-creating-iam-azure-stack-hub.adoc#manually-creating-iam-azure-stack-hub[manual mode]. Manual mode is used in environments where the cloud IAM APIs are not reachable.
```

Comment:
I don't think this is really a prerequisite because we will later create the install config. This link creates the install config and then the secrets, so it can't really happen before these steps.

Reply:
Agreed. I have added Manually manage cloud credentials before the step for deploying the cluster.

Reply:
@codyhoag The entire section on Manually creating IAM for Azure Stack Hub is technically accurate as written. However, in my opinion, the info is out of context of an actual installation. The relevant steps to generate the manifests have been added to the installation assembly and the respective module conditionalized accordingly, so as not to break usage elsewhere. I am proposing that the entire Manually creating IAM for Azure Stack Hub be removed from the TOC. The IPI rewrite gets the information to the user when they need with the appropriate level of detail. The current UPI doc addresses this in step 7 [1]

WDYT about removing Manually creating IAM for Azure Stack Hub?

file:///home/mpytlak/openshift-docs/_preview/openshift-enterprise/osdocs-2950-default/installing/installing_azure_stack_hub/installing-azure-stack-hub-user-infra.html#installation-user-infra-generate-k8s-manifest-ignition_installing-azure-stack-hub-user-infra

Reply:
@codyhoag What do you think about removing Manually creating IAM for Azure Stack Hub? ^^^

Reply:
Yeah, I agree it makes sense to remove it as a top level prereq.

In terms of deleting the section as a whole (not sure if this is what you meant), I would vote to keep it in the left nav as a section for consistency. Most providers have this section, and it details several other informational topics on the CCO, like upgrading credentials and other modes (which this section would explain once those are supported for ASH).

I think this info could be reorganized to better integrate into the installation, or extract it to a more centralized location where all providers can reference it, but that might be a good follow-up task.

Reply:
> Yeah, I agree it makes sense to remove it as a top level prereq.
> 
> In terms of deleting the section as a whole (not sure if this is what you meant), I would vote to keep it in the left nav as a section for consistency. Most providers have this section, and it details several other informational topics on the CCO, like upgrading credentials and other modes (which this section would explain once those are supported for ASH).
> 
> I think this info could be reorganized to better integrate into the installation, or extract it to a more centralized location where all providers can reference it, but that might be a good follow-up task.

To tie this off, Cody and I met and walked through my proposed changes. We agreed that the relevant material does appear in the context of the assembly and Manually creating IAM for Azure Hub can be removed until such time that ASH supports additional CCO modes of operation.